dijet/selection/lepton_selection.py

```diff
@@ -24,16 +24,17 @@ def lepton_selection(
     # lepton selection based on old UHH2 framework
     # https://github.com/UHH2/DiJetJERC/blob/ff98eebbd44931beb016c36327ab174fdf11a83f/src/AnalysisModule_DiJetTrg.cxx#L703
     # IDs in JME Nano https://cms-nanoaod-integration.web.cern.ch/integration/master-106X/mc102X_doc.html
-    # mask for muons
+    events = attach_coffea_behavior(events, collections={"Muon": { "type_name": "Muon","skip_fields": "*Idx*G",}})
+    # from IPython import embed; embed()
```

Comment:
Would remove the commented-out debugger, but maybe keep the old comment about the muon mask
```suggestion
    events = attach_coffea_behavior(events, collections={"Muon": { "type_name": "Muon","skip_fields": "*Idx*G",}})
    # mask for muons
```